src/content/docs/it/core-concepts/astro-pages.mdx

```diff
@@ -0,0 +1,90 @@
+
+📚 Ulteriori informazioni sul [routing in Astro](/it/core-concepts/routing/).
+
+### Collegamento tra le pagine
+
+Scrivi l´elemento standard HTML [`<a>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) nelle tue pagine Astro per collegarti ad altre pagine del tuo sito.
+
+## Pagine Astro
+
+Le pagine Astro utilizzano l'estensione del file `.astro` e supportano le stesse funzionalità di [Componenti Astro](/it/core-concepts/astro-components/).
+
+```astro
+---
+// Example: src/pages/index.astro
+---
+<html lang="en">
+  <head>
+    <title>My Homepage</title>
+  </head>
+  <body>
+    <h1>Welcome to my website!</h1>
+  </body>
+</html>
+```
+
+Per evitare di ripetere gli stessi elementi HTML su ogni pagina, puoi spostare gli elementi comuni `<head>` e `<body>` nei tuoi [componenti di layout](/it/core-concepts/layouts/). Puoi utilizzare tutti i componenti di layout che desideri.
+
+```astro {3} /</?MySiteLayout>/
+---
+// Example: src/pages/index.astro
+import MySiteLayout from '../layouts/MySiteLayout.astro';
+---
+<MySiteLayout>
+  <p>My page content, wrapped in a layout!</p>
+</MySiteLayout>
+```
+
+📚 Leggi di più sui [componenti del layout](/it/core-concepts/layouts/) in Astro.
+
+## Pagine Markdown/MDX
+
+Astro tratta anche qualsiasi file Markdown (`.md`) all'interno di `src/pages/` come pagine del tuo sito web finale. Se hai [l´integrazione MDX installata](/it/guides/integrations-guide/mdx/#installation), tratta anche i file MDX (`.mdx`) allo stesso modo. Questi sono comunemente usati per pagine ricche di testo come post di blog e documentazione.
+
+[Le raccolte di contenuti delle pagine Markdown o MDX](/it/guides/content-collections/) in `src/content/` possono essere utilizzate per [generare pagine dinamicamente](/it/core-concepts/routing/#dynamic-routes).
+
+I layout di pagina sono particolarmente utili per i [file Markdown](#pagine-markdownmdx). I file Markdown possono utilizzare la speciale proprietà frontmatter `layout` per specificare un [componente layout](/it/core-concepts/layouts/) che avvolgerà il contenuto Markdown in un completo documento di pagina, `<html>...</html>`.
+
+```md {3}
+---
+# Example: src/pages/page.md
+layout: '../layouts/MySiteLayout.astro'
+title: 'My Markdown page'
+---
+# Title
+
+This is my page, written in **Markdown.**
+```
+
+📚 Ulteriori informazioni su [Markdown](/it/guides/markdown-content/) in Astro.
+
+## Pagine HTML
+
+I file con estensione `.html` possono essere inseriti in `src/pages/` e utilizzati direttamente come pagine del tuo sito. Tieni presente che alcune funzionalità chiave di Astro non sono supportate in [Componenti HTML](/it/core-concepts/astro-components/#componenti-html).
+
+## Pagina di errore 404 personalizzata
+
+Per una pagina di errore 404 personalizzata, puoi creare un file `404.astro` o `404.md` in `/src/pages`.
+
+Questo creerà una pagina `404.html`. La maggior parte dei [servizi di distribuzione](/it/guides/deploy/) la troveranno e la utilizzeranno.
```

Comment:
I'm not sure about `servizi di distribuzione`, usually in Italian we use this term for gas companies.
I would use something else, like `fornitori di hosting` or `fornitori di servizi cloud`. Any other idea?

Reply:
Yeah, we could merge your two suggestions: `fornitori di servizi di hosting`.

```suggestion
Questo creerà una pagina `404.html`. La maggior parte dei [fornitori di servizi di hosting](/it/guides/deploy/) la troveranno e la utilizzeranno.
```